spec/features/users/user_creates_account_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
+
+feature 'user creates an account' do
+
+  before :each do
+    visit root_path
+    click_link "Sign Up"
+  end
+
+  scenario "user specifies valid and required info as a volunteer" do
+    user = FactoryGirl.create(:user)
+    # attach_file "Profile Photo",
+      #"#(Rails.root)/spec/support/images/soldier.jpg"
+    fill_in "First Name", with: user.first_name
+    fill_in "Last Name", with: user.last_name
+    fill_in "Email", with: "user@example.com"
+    fill_in "Password", with: user.password
+    fill_in "Password Confirmation", with: user.password
+    select 'Volunteer', from: "Are you a volunteer or an organization?"
+
+    click_button "Sign Up"
+
+    expect(page).to have_content("Welcome")
+    expect(page).to have_content("Sign Out")
+    # expect(page).to have_css("img[src*='soldier.jpg']")
+  end
+
+  # scenario "user uploads non-image file" do
+  #
+  # end
+
+  # scenario "user specifies valid and required info as an organization" do
+  #   org_user = FactoryGirl.create(:organization)
+  #   # attach_file "Profile Photo",
+        #"#(Rails.root)/spec/support/images/soldier.jpg"
```

Comment:
Incorrect indentation detected (column 8 instead of 2).<br>Missing space after #.